lib/requests.py

```diff
@@ -57,36 +59,40 @@ def set_completion_state(
     net_length = len(substitution)
     request = {
         "Selection": {
-            "language": [x for x in file_name.split(".")][-1],
+            "language": os.path.splitext(file_name)[1][1:],
```

Comment:
I have no idea what it does now (I prefer the previous one). I wanted you to supply a static file extension (something like `UNKNOWN` if there is no file extension.